tests/cypress/tests/standalone/edit_cell_test.ts

```diff
@@ -4,6 +4,34 @@ import Key from 'cypress/Key';
 
 import { AppMode, ReadWriteModes } from 'demo/AppMode';
 
+Object.values([
+    ...ReadWriteModes,
+    AppMode.TaleOfTwoTables
+]).forEach(mode => {
+    describe(`edit, mode=${mode}`, () => {
+        beforeEach(() => {
+            cy.visit(`http://localhost:8080?mode=${mode}`);
+            DashTable.toggleScroll(false);
+        });
+
+        // https://github.com/plotly/dash-table/issues/50
+        it('can edit on "enter"', () => {
+            DashTable.getCell(0, 1).click();
+            cy.wait(1000);
+            DOM.focused.type(`abc${Key.Enter}`);
+            DashTable.getCell(0, 1).within(() => cy.get('.dash-cell-value').should('have.html', `abc`));
+        });
+
+        it('can edit when clicking outside of cell', () => {
+            DashTable.getCell(0, 1).click();
+            DOM.focused.type(`abc`);
+            cy.wait(1000);
```

Comment:
The delay is not great but need to guarantee the table has had time to re-render if stuck in a loop.

Reply:
Hmm OK. I guess if it's an async system causing the infinite loop that makes sense. Please put a comment by each of these waits referencing this PR so we know why it's there and we don't try to optimize it away later, since presumably the test will still pass if it were removed - it just wouldn't be testing what it's supposed to be testing.